src/Files.App/UserControls/PathBreadcrumb.xaml

```diff
@@ -100,6 +100,8 @@
 								<MenuFlyout.MenuFlyoutPresenterStyle>
 									<Style TargetType="MenuFlyoutPresenter">
 										<Setter Property="MaxHeight" Value="400" />
+										<!--  Workaround for https://github.com/files-community/Files/issues/13078  -->
+										<Setter Target="HighContrastAdjustment" Value="None" />
```

Comment:
Can we base this on the default style or would that cause other styling issues?

Reply:
I tried basing default style on this, but it did not worked.

https://github.com/files-community/Files/pull/14664/files#diff-dc0184eeec8db43468bd55586c4362feb47f4fa7a2af809f828f42c5cb1eeb5aR34-R37 should effect on this because it targets the same type(MenuFlyoutPresenter).

Reply:
Can you share a before and after screenshot?

Reply:
I'm away from my PC rn but I'm sure I can say it's the same visual improvement as other flyout ones.